### PR TITLE
[cli] remove spec build code_origin and origin_filename'

### PR DESCRIPTION
**Pull request by aviaIguazio**

### CODE REVIEW

File: cli/item_to_function.py [hunk 1]
```diff
@@ -188,6 +200,8 @@ def create_function_yaml(
 
     function_object.export(target=str(output_path.resolve()))
 
+    remove_build_info_from_yaml(function_path=output_path)
```
Comment (theSaarco) [inline cli/item_to_function.py:203]: Why do this *after* you export the yaml file? Better to just remove the values from `function_object` before exporting it...

File: cli/item_to_function.py [hunk 2]
```diff
@@ -205,8 +219,21 @@ def bump_function_yaml_version(item_path: Path):
...
+    function_yaml['spec']['build']['origin_filename'] = ''
+    with open(str(function_path), 'w') as file:
+        yaml.safe_dump(function_yaml, file, default_flow_style=False)
+
+
+
+
+
+
+
```
Comment (theSaarco) [inline cli/item_to_function.py:234]: Why all these spaces?

File: cli/item_to_function.py [hunk 3]
```diff
@@ -117,6 +117,18 @@ def _get_item_yaml(item_path: Path) -> dict:
...
+        if (function_path / "function.yaml").exists():
+            function_path = function_path / "function.yaml"
+        else:
+            raise FileNotFoundError(f"{function_path} does not contain a item.yaml file")
+    elif not function_path.exists():
+        raise FileNotFoundError(f"{function_path} not found")
+
+    item_yaml = full_load(open(function_path, "r"))
+    return function_path, item_yaml
+
```
Comment (theSaarco) [inline cli/item_to_function.py:131]: Looks like this is not used, why is this here?

**Review by theSaarco — CHANGES_REQUESTED** [review]
I think this should be done differently - just modify the object prior to exporting it, not export and then re-write the yaml.

File: cli/item_to_function.py (hunk 1 above)
Comment (aviaIguazio) [reply cli/item_to_function.py:203]: i agree in order to do that we need to change 
https://github.com/mlrun/mlrun/blob/development/mlrun/runtimes/base.py
this a workaround

File: cli/item_to_function.py (hunk 3 above)
Comment (aviaIguazio) [reply cli/item_to_function.py:131]: will remove it

File: cli/item_to_function.py (hunk 2 above)
Comment (aviaIguazio) [reply cli/item_to_function.py:234]: will remove it

File: cli/item_to_function.py (hunk 1 above)
Comment (theSaarco) [reply cli/item_to_function.py:203]: Why change the export? You can just go and do `function_object.spec.build.code_origin = ''` (and the same for `origin_filename`). Doing this here, rather than inside the `export()` actually makes sure you don't touch any of the regular MLRun flows, but rather just override the behavior for the hub purposes.

Comment (aviaIguazio) [reply cli/item_to_function.py:203]: ok fixed in the new PR
